public/video-ui/src/reducers/rootReducer.js

```diff
@@ -1,11 +1,13 @@
 import { combineReducers } from 'redux';
+import config from './configReducer';
 import error from './errorReducer';
 import video from './videoReducer';
 import videos from './videosReducer';
 import { reducer as form } from 'redux-form'
 
 
 export default combineReducers({
```

Comment:
Just for my own curiosity is it coincidence that these are in alphabetical order or is this good style for reducer combining?